amundsen_application/static/js/components/common/ResourceListItem/DashboardListItem/index.tsx

```diff
@@ -53,15 +54,16 @@ class DashboardListItem extends React.Component<DashboardListItemProps, {}> {
             { getSourceDisplayName(dashboard.product, dashboard.type) }
           </div>
           <div className="resource-badges">
-            {
-               dashboard.last_successful_run_timestamp &&
-               <div>
-                 <div className="title-3">{ Constants.LAST_RUN_TITLE }</div>
-                 <div className="body-secondary-3">
-                   { formatDate({ epochTimestamp: dashboard.last_successful_run_timestamp }) }
-                 </div>
+             <div>
+               <div className="title-3">{ Constants.LAST_RUN_TITLE }</div>
+               <div className="body-secondary-3">
+                 {
+                   dashboard.last_successful_run_timestamp ?
+                   formatDate({ epochTimestamp: dashboard.last_successful_run_timestamp }) :
+                   NO_TIMESTAMP_TEXT
+                 }
                </div>
-             }
+             </div>
```

Comment:
Is this extra div necessary for a styling thing?

Reply:
Yes. In a component where the items are laid out horizontally, this div wraps the title and the timestamp so they stack vertically as a chunk. 

It could technically be removed if we created and applied a few styles here related to `DashboardListItem` only, but for now we have this approach of sharing the exact styles defined in the parent directory that are shared across all the list items.